extension/source/QuillPage/components/ViewSecretPhrasePanel.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { browser } from 'webextension-polyfill-ts';
+import Button from '../../components/Button';
+import Range from '../../helpers/Range';
+import QuickColumn from './QuickColumn';
+import QuickRow from './QuickRow';
+
+const ViewSecretPhrasePanel: React.FunctionComponent = () => {
+  const [expanded, setExpanded] = React.useState(false);
+
+  const classes = ['view-secret-phrase-panel'];
+
+  if (expanded) {
+    classes.push('expanded');
+  }
+
+  const secretWords = [
+    'Potato',
+    'Velvet',
+    'Keen',
+    'Water',
+    'Travel',
+    'Pill',
+    'Book',
+    'Photo',
+    'Image',
+    'Space',
+    'Pause',
+    'Power',
+  ];
+
+  return (
+    <div className={classes.join(' ')}>
+      <div className="instructions-text">
+        <h3>
+          Congratulations!
+          <br />
+          You have created a wallet.
+        </h3>
+        <p>
+          Below is your secret recovery phrase, which you will need when
+          restoring your wallets should you lose access.
+        </p>
+      </div>
+      <div className="secret-phrase-box">
+        {Range(4).map((i) => (
+          <QuickRow key={`row${i}`}>
+            {Range(3).map((j) => (
+              <QuickColumn key={`column${j}`}>
+                {3 * i + j + 1}. {secretWords[3 * i + j]}
+              </QuickColumn>
+            ))}
+          </QuickRow>
+        ))}
+      </div>
```

Comment:
```suggestion
      {expanded && <div className="secret-phrase-box">
        {Range(4).map((i) => (
          <QuickRow key={`row${i}`}>
            {Range(3).map((j) => (
              <QuickColumn key={`column${j}`}>
                {3 * i + j + 1}. {secretWords[3 * i + j]}
              </QuickColumn>
            ))}
          </QuickRow>
        ))}
      </div>}
```
This will prevent rendering the box altogether when hidden. Right now its just hidden.

Reply:
Does it matter though? [Discussion link.](https://github.com/jzaki/bls-wallet/pull/93/files#r784378578)

Reply:
Done.